bootstrap/patches/lodestar-status.yaml

```diff
@@ -20,7 +20,7 @@ spec:
           lodestar-git-api: v1.2.1
           lodestar-status: v1.1.0
           lodestar-config: v1.0.0
-          resource-dispatcher: v1.0.13
+          resource-dispatcher: v1.0.14
```

Comment:
This should be part of a release. Let's not update as part of this clean-up.

Reply:
The thing is, resource-dispatcher is already on v1.0.14 - and we got disrepancy between LodeStar Status service VS what's actually running on cluster.